README.md

```diff
@@ -56,6 +56,14 @@ cd ambianic-ui
 npm install
 ```
 
+### Enviornment Variables
+To use the notifications feature, the PWA requires the `VUE_APP_FUNCTIONS_ENDPOINT` which should point to the [Cloud API](https://github.com/ambianic/ambianic-subscriptions) either running in production or locally.
+
+```
+VUE_APP_FUNCTIONS_ENDPOINT=<CLOUD_API_ENDPOINT>/.netlify/functions
```

Comment:
@vickywane it's good to have this doc section. Let's remove VUE from the variable name. It does not matter if we use react or vue, the endpoint for cloud API will be the same.

Reply:
We need to have the `VUE_APP` prefixed to the variable name for value-loaded automatically without using `dotenv`.